ZEP/instructions/zep0000.md

```diff
@@ -0,0 +1,399 @@
+a narrative rationale for the specification changes.
+
+We intend ZEPs to be the primary mechanism for evolving the spec, collecting
+community input on major issues and documenting the design decision that has
+gone into Zarr. In addition, the ZEP author is responsible for building
+consensus within the community and documenting dissenting opinions.
+
+Because the ZEPs are maintained as text files in a versioned repository, their
+revision history is the historical record of the feature proposal.
+
+## ZEP audience
+
+The typical primary audience for ZEPs is the developers working on Zarr and its
+various implementations, the Zarr steering council as well as the Zarr
+community.
+
+The broader Zarr community may also choose to use the process to document
+expected feature additions and to manage complex design coordination problems
+that require collaboration across multiple projects.
+
+## ZEP types
+
+> Specification ZEP
+
+Specification ZEPs deal with the changes related to [Zarr Specifications].
+The SPEC ZEP (abbreviation of Specification ZEP) introduces changes in the core
+specification of Zarr. The changes related to core specification follow a
+strict and thorough review process and should be adopted by everyone in the
+Zarr community.
+
+- Core protocol ZEP
+
+Describes a ZEP which involves changes in the core specification of Zarr. Core
+protocol ZEPs (commonly known as Core ZEPs) are a part of SPEC ZEPs and apply to
+[Zarr Specifications] and its various implementations under the
+[zarr_implementations] GitHub repo. The core protocol ZEPs should be adopted by
+every implementation of Zarr and, in general, the overall Zarr community. Core
+ZEPs must go through a thorough review process, including involvement,
+discussion, and voting from the author of the proposed ZEP, Zarr Steering
+Council, author(s) of various Zarr implementations, and open-source
+projects/research groups using Zarr and the general Zarr community. The general
+advice is that everyone should be made aware of changes introduced by core
+ZEPs. Core protocol ZEPs require community consensus, and consumers are typically not
+free to ignore them.
+
+Note: Currently, Specification ZEPs only deal with the core storage
+specification of Zarr. The scope of SPEC ZEPs will be extended in the future.
+
+> Informational ZEP
+
+Describes a ZEP design issue or provides general guidelines or information to
+the Zarr community but does not propose a new feature. Informational ZEPs do
+not necessarily represent Zarr community consensus or recommendation, so users
+and implementers are free to ignore informational ZEPS or follow their advice.
+
+> Process ZEP
+
+Describes a new process around Zarr and its implementations or proposes a
+change to (or an event in) a process. They may propose an implementation, but
+not to Zarr’s codebase; they require community consensus; unlike informational
+ZEPs, they are more than recommendations, and users are typically not free to
```

Comment:
What do Zarr users care about our process? Surely we are talking about developers here, no?

```suggestion
ZEPs, they are more than recommendations, and Zarr developers are typically not free to
```